### DISCUSSION TITLE
.GIF support

### DISCUSSION THREAD

**Discussion by EriveltoLuna** [mathoudebine/turing-smart-screen-python#16] [General]
I was wondering if it supports .gif files, i'm new to programming, have no idea how this script works but i'm willing to learn

**Comment by mathoudebine** [comment]
Hello @EriveltoLuna 
Unfortunately the program does not support gif.
This is a hardware issue : the screen refresh rate is very slow (takes approx. 4 seconds to load an image full screen) so displaying a gif would not be very responsive
Even if you display a gif to a small part of the screen, it would still take some time between each frame (~1s if the gif takes 1/3 of the screen) so i think the display is not made for it ☹️

**Reply by EriveltoLuna** [reply]
Thanks for the answer, it really takes a long time to load an image
